packages/ui/src/components/Activity/ActivitySummaryMessage.tsx

```diff
@@ -35,19 +40,30 @@ function SummaryMessageRaw({
       <>
         <ContactName
           fontSize="$s"
+          lineHeight={18}
```

Comment:
@dnbrwstr We don't have a value for this in the Tamagui config  and we only use it in this view.